memgpt/server/rest_api/utils.py

```diff
@@ -17,30 +17,10 @@
 def sse_formatter(data: Union[dict, str]) -> str:
     """Prefix with 'data: ', and always include double newlines"""
     assert type(data) in [dict, str], f"Expected type dict or str, got type {type(data)}"
-    data_str = json_dumps(data) if isinstance(data, dict) else data
```

Comment:
The refactored `json_dumps` shouldn't be used here since we need compact JSON with no indents for the POST SSE responses